packages/colorpickers/src/elements/ColorSwatchDialog/index.tsx

```diff
@@ -116,11 +127,21 @@ export const ColorSwatchDialog = forwardRef<
       controlledSelectedColor = colors[selectedRowIndex][selectedColIndex];
     }
 
+    const openDialog = () => {
+      setReferenceElement(buttonRef.current);
+      onDialogChange && onDialogChange({ isOpen: true });
+    };
+
+    const closeDialog = () => {
+      setReferenceElement(null);
+      onDialogChange && onDialogChange({ isOpen: false });
+    };
+
```

Comment:
Should this same API be reflected in `ColorpickerDialog`? As a consumer, I would expect parity.